lookups/lookup-get-basic-example-1/lookup-get-basic-example-1.5.x.rb

```diff
@@ -12,10 +12,10 @@
 
   puts number.carrier['type']
   puts number.carrier['name']
-rescue Twilio::REST::RestError => err
-  if err.status_code === 404
+rescue Twilio::REST::RestError => e
+  if e.status_code === 404 # rubocop:disable Style/CaseEquality
```

Comment:
I think this comment would be shown on the pages using this example, maybe w should rewrite the code to avoid this comment?

Reply:
Yes you are right I missed that detail, I removed that comments.